Form/Extension/Field/Type/FormTypeFieldExtension.php

```diff
@@ -115,13 +115,12 @@ public function buildView(FormView $view, FormInterface $form, array $options)
                 'class' => false,
                 'options' => $this->options,
             );
+            $view->vars['sonata_help'] = $sonataAdminHelp;
```

Comment:
Why not just write `$sonataAdminHelp = isset($options['sonata_help']) ? $options['sonata_help'] : null;`?

Reply:
The assignment you are talking about is at the very top of the method definition and $sonataAdminHelp is used two times in the method, so I moved in to the top.

Reply:
Oh ok didn't see the second occurence.